facia/app/controllers/FaciaController.scala

```diff
@@ -25,6 +25,14 @@ trait FaciaController extends Controller with Logging with ExecutionContexts wit
 
   val frontJsonFapi: FrontJsonFapi
 
+  private def getEditionFromString(edition: String) = {
+    val editionToFilterBy = edition match {
+      case "international" => "int"
+      case _ => edition
+    }
+    Edition.all.find(_.id.toLowerCase() == editionToFilterBy).getOrElse(Edition.all.head)
+  }
+
```

Comment:
Just exposing this function so it can be reused both in `renderSomeFrontContainers` and `renderSomeFrontContainersMf2`